tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/swtbot/PitResultNotifier.java

```diff
@@ -138,7 +138,7 @@ public boolean test() {
                 public String getFailureMessage() {
                     return "No summary was generated after the specified time.";
                 }
-            }, (timeOut > 0) ? timeOut : SWTBotPreferences.TIMEOUT);
+            }, (timeOut > 0) ? timeOut : Long.parseLong(System.getProperty(SWTBotPreferences.KEY_TIMEOUT)));
```

Comment:
In SWTBot you have

```java
public static long TIMEOUT = toLong(System.getProperty(KEY_TIMEOUT, "5000"), 5000);
```

so please, just rever this change, which is only confusing and completely equivalent

Reply:
I did not know that.